ui/widgets/resizable.js

```diff
@@ -131,11 +132,16 @@ $.widget( "ui.resizable", $.ui.mouse, {
 
 			this.elementIsWrapper = true;
 
+			marginTop = this.originalElement.css( "marginTop" );
```

Comment:
This could be DRYed out a bit by creating an object here with the margins and using it for the two calls below.